boxoffice/models/line_item.py

```diff
@@ -73,6 +75,38 @@ def calculate(cls, line_item_dicts, coupons=[]):
             line_items.extend(item_line_items[item_id])
         return line_items
 
+    @classmethod
+    def coupon_used_count(cls, coupon):
+        count_query = select([func.count(LineItem.discount_coupon_id)]).where(LineItem.discount_coupon_id == coupon.id)
+        result = db.session.execute(count_query)
+        return result.first()[0]
```

Comment:
Why is this classmethod here? This should be an instance method on DiscountCoupon.